src/Java.Interop/Java.Interop/JavaObjectArray.cs

```diff
@@ -164,7 +164,7 @@ internal override bool TargetTypeIsCurrentType (Type? targetType)
 
 		internal sealed class ValueMarshaler : JniValueMarshaler<IList<T>> {
 
-			public override IList<T> CreateGenericValue (ref JniObjectReference reference, JniObjectReferenceOptions options, Type? targetType)
+			public override IList<T> CreateGenericValue (ref JniObjectReference reference, JniObjectReferenceOptions options, [DynamicallyAccessedMembers (ConstructorsAndInterfaces)] Type? targetType)
```

Comment:
This is long enough that each parameter should get its own line.  Parameters are indented 2 tab stops:

```csharp
public override IList<T> CreateGenericValue (
        ref JniObjectReference reference,
        JniObjectReferenceOptions options,
        [DynamicallyAccessedMembers (ConstructorsAndInterfaces)]
        Type? targetType)
{
    // …
}
```